src/docfx/models/tableOfContents/TableOfContentsItemModel.cs

```diff
@@ -0,0 +1,11 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Docs.Build
+{
+    internal class TableOfContentsItemModel
```

Comment:
Needs design discussion:

- Is this the input model or output model, or could they be just one
- Can `TocItemModel` and `TocModel` be consolidated into one model

Reply:
its output model, just like markdown, we don't have input model


---
In reply to: [184585500](https://github.com/dotnet/docfx/pull/2696#discussion_r184585500) [](ancestors = 184585500)

Reply:
**input** means the schema of `toc.yml`, **output** means the schema of `toc.json for publishing

Reply:
it can be toc.md, actually not clear about the schema is only for about SDP or for all?

---
In reply to: [184595574](https://github.com/dotnet/docfx/pull/2696#discussion_r184595574) [](ancestors = 184595574)